### DISCUSSION TITLE
How to upgrade to latest version?

### DISCUSSION THREAD

**Discussion by mahdijafaridev** [rappasoft/laravel-livewire-tables#555] [Q&A] [answered]
I have installed v1.13 and I need to use the new feature recently added, like: footer(), linkTo(), and ...
I couldn't find any command to upgrade the package to the latest version.

**Comment by rappasoft** [comment, marked as answer]
They aren't major releases so composer should be able to do it.

**Reply by mahdijafaridev** [reply]
I tried `composer update` and it didn't update the version in `composer.json` file. My problem is that new features don't work for me. and I need them. I manually changed 1.13 to 1.20, and again those new functions don't work. any suggestions?

**Reply by rappasoft** [reply]
You should just leave your version as `^1.0` in composer. Make sure you don't have any package vendor views files after you update.

**Reply by mahdijafaridev** [reply]
Thanks. do you mean that at `/resources/views/livewire-tables/rows` shouldn't be any file?

I searched in vendor files of rappasoft, and there was `hasFooter()` function but no `linkTo()`.

**Reply by rappasoft** [reply]
The files under resources/views/vendor/laravel-livewire-tables

But if you searched the vendor files and didn't see the functionality then you're not on the latest version.

**Reply by mahdijafaridev** [reply]
Thank you so much. The problem is solved
